### PR TITLE
feat: Align iOS version (SDKCF-5011)

### PR DESCRIPTION
**Pull request by malayli**

### CODE REVIEW

**Review by imaginaris — CHANGES_REQUESTED** [review]
@malayli please also refactor `./Sources/RSDKUtilsMain/KeyStore.swift:79`. The `if #available(iOS 11.3, *)` is not needed anymore

**Review by apparition47 — APPROVED** [review]

File: RSDKUtils.podspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.homepage     = "https://github.com/rakutentech/ios-sdkutils"
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.source       = { :git => "https://github.com/rakutentech/ios-sdkutils.git", :tag => s.version.to_s }
-  s.platforms    = { :ios => '11.0', :watchos => '6.0' }
+  s.platforms    = { :ios => '12.0', :watchos => '6.0' }
```
Comment (donnie-jp) [inline RSDKUtils.podspec:9]: This is a breaking semver change so next version will need to be major.

**Review by donnie-jp — APPROVED** [review]

File: RSDKUtils.podspec (hunk 1 above)
Comment (malayli) [reply RSDKUtils.podspec:9]: Do we need to create a ticket to release a new version of RSDKUtils? @donnie-jp

Comment (donnie-jp) [reply RSDKUtils.podspec:9]: yes @malayli 

you can ask PJM/PDM to support that.

Comment (imaginaris) [reply RSDKUtils.podspec:9]: @donnie-jp We've already broken backward compatibility in previous commits

**Pull request merged**